### DISCUSSION TITLE
About Custom Block style

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1121] [Questions and Help]
Hi, @SamTolmay 
I introduced the Ant Design style into the Custom Block, but this style will overwrite the Lowdefy style. Is there any way to solve it?
For example, the style of Button Block is overwritten:
![image](https://user-images.githubusercontent.com/72787010/154640308-f96d9a9e-3bd2-4883-bb9a-5bd1ef34dd46.png)
Rate.js
```
import React, {useState} from 'react';
import { Rate } from 'antd';
import 'antd/dist/antd.css';
const RateCompontents = (props) => {
    const { blockId, methods, properties, rename } = props.props
    const { data, readOnly } = properties
    const [val, setVal] = useState(data || '')
    const changeVal = (val) => {
        console.log(val);
        methods.triggerEvent({ name: 'onChange',event: {value: val} })
        return setVal(val)
    }

    return (
        <Rate onChange={(val) => changeVal(val)} value={val} id={blockId} disabled={readOnly} />
    )
}

export default RateCompontents;

```

**Comment by SamTolmay** [comment]
Hi @xiaoxiji1234 

What version of Antd are you using for the custom block? We are still using `4.4.2` on version 3. It would probably be best if you also use that version.

You probably also don't need to import the css (`import 'antd/dist/antd.css'`) since this css is aready part of the Lowdefy bundle. 

This will change in v4, where we import individual styles for the antd blocks, but that will be part of the migration documentation.

**Comment by xiaoxiji1234** [comment]
Hi, @SamTolmay 
The Antd version of the custom block is `4.16.13`, and I tried to downgrade it to `4.4.2`. Because my custom block is developed based on Antd components. If  I don't need to import the css  `import 'antd/dist/antd.css'` this can cause a lot of style problems.
